project-keeper/src/main/java/com/exasol/projectkeeper/sources/analyze/generic/ProcessResult.java

```diff
@@ -0,0 +1,67 @@
+package com.exasol.projectkeeper.sources.analyze.generic;
+
+import java.util.Objects;
+
+/**
+ * Result of executing a process.
+ */
+public final class ProcessResult {
+    private final String outputStreamContent;
```

Comment:
Should we add a field to return code?

Reply:
The return code is always zero. If the process fails with non-zero code it will throw an exception.